tests/main/auto-refresh/task.yaml

```diff
@@ -10,29 +10,32 @@ execute: |
     echo "Install a snap from stable"
     snap install test-snapd-tools
     snap list | MATCH 'test-snapd-tools +[0-9]+\.[0-9]+'
+    snap set core refresh.schedule="0:00-$(date +%H:%M:%S --date=30seconds)"
```

Comment:
We don't take seconds anymore, I think?  Also not clear what the goal is here. This probably shouldn't be working as is, so perhaps unnecessary?